### PR TITLE
Update truenas-scale.mdx

### PR DESCRIPTION
**Pull request by wolf4914**
The sequence here was a bit messed up so I swapped # 7 and #8

**Description**

⚒️ Fixes  # The sequence here was a bit messed up so I swapped # 7 and #8

**⚙️ Type of change**
Sequence of steps

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning
- [ ] I made sure the title starts with `feat(chart-name):`, `fix(chart-name):` or `chore(chart-name):`

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

**Comment by Ornias1993** [comment]
Please make sure CLI is signed and tests are passing

**Comment by bitpushr** [comment]
@wolf4914

File: src/content/docs/clustertool/virtual-machines/truenas-scale.mdx [hunk 1]
```diff
@@ -52,10 +52,10 @@ The Network Interface should then look like this:
 
   6. Enter the IP address and Subnet mask you previously took note of
 
-  7. Click `Test Changes` and confirm the prompt. Wait a bit for the UI to become responsive again
-    ![Network Changes ](./img/network_changes.png)
+  7.Click on `Save Changes` to save the changes you have made
 
-  8. Click on `Save Changes` to save the changes you have made
+  8. Click `Test Changes` and confirm the prompt. Wait a bit for the UI to become responsive again
+    ![Network Changes ](./img/network_changes.png)
```
Comment (stavros-k) [inline src/content/docs/clustertool/virtual-machines/truenas-scale.mdx:58]:
```suggestion
    ![Network Changes](./img/network_changes.png)
```

**Comment by PrivatePuffin** [comment]
@bitpushr If no response in 2 weeks he will get a repo ban for time wasting.

**Pull request closed without merging**